WcaOnRails/app/assets/stylesheets/application.css.scss

```diff
@@ -14,8 +14,9 @@
  */
 
 @import "variables";
-@import "bootstrap-sprockets";
-@import "bootstrap";
+@import "bootstrap_and_overrides";
+//@import "bootstrap-sprockets";
```

Comment:
The bootstrap_and_overrides is being (currently) used for the table-striped colors.

So maybe I'll remove it from this PR.